README.md

```diff
@@ -1,47 +1,121 @@
-# Use and deploy this stack
+# Docker reference stack
 
-To run it, you need Docker >= 1.10, and docker-compose
+This repository provides a [Docker](https://www.docker.com/)-based technology stack, comprised of many components which together emulate a research application and an electronic health record vendor. These components can be used as reference implementations for developers who wish to develop a research application that consumes electronic health record data from participants, or for EHR vendors who are exploring implementing the [SMART Health IT platform](https://smarthealthit.org/).
 
-## Clone this repo with submodules
- * `git clone --recursive https://github.com/sync-for-science/reference-stack-docker`
+## Getting started
 
-## Update submodules after git pull
- * `git submodule update --init --recursive`
+### Cloning the repository
 
-# Pull or build
+The stack makes heavy use of [git submodules](https://git-scm.com/docs/git-submodule), so one extra step is required:
 
-Run either one of the following to build the images yourself, or pull them from Docker Hub:
+    $ git clone https://github.com/sync-for-science/reference-stack-docker
```

Comment:
Should we recommend a `git clone --recursive` in the first place? Pros/cons?

Reply:
Ah, this makes a lot of sense - I wasn't aware of that option. Do you see any cons?

Reply:
None I've encountered. (One note, since we added the Imaging repos as git submodules, is that the imaging refernce server has a _huge_ repo histtory, which slows the whole process down. We might experiment with `--recursive --shallow-submodules` to avoid pulling in the entire submodule history.)

Reply:
It seems that `--shallow-submodules` fails if the submodules are checked out to a specific hash rather than a branch or tag 😞 

Checking out everything recursively takes about 40s on my machine - is it significantly different for yours?